src/org/scada_lts/ds/reactivation/ReactivationManager.java

```diff
@@ -44,6 +44,13 @@ public ReactivationManager() {
         }
     }
 
+    protected void removeInfoAboutJob(String keyNameJob) {
+
+        Map.Entry<String,Integer> dsInfo = sleepDsIndexJobName.get(keyNameJob);
+        sleepDsIndexIdDs.remove(dsInfo.getValue());
```

Comment:
Is it possible to change name from :

sleepDsIndexIdDs.remove(idDs);

to

sleepDataSourceIndexIdDataSource

?

Reply:
improved